src/DataViews/LiveFunctionsDataViewTest.cpp

```diff
@@ -87,6 +92,41 @@ constexpr int kColumnModule = 8;
 constexpr int kColumnAddress = 9;
 constexpr int kNumColumns = 10;
 
+constexpr size_t kNumThreads = 2;
+constexpr std::array<uint32_t, kNumThreads> kThreadIds = {111, 222};
+const std::array<std::string, kNumThreads> kThreadNames = {"Test Thread 1", "Test Thread 2"};
+
+constexpr size_t kNumTimers = 3;
+constexpr std::array<uint64_t, kNumTimers> kStarts = {1000, 2345, 6789};
+constexpr std::array<uint64_t, kNumTimers> kEnds = {1500, 5432, 9876};
+constexpr std::array<uint64_t, kNumTimers> kThreadIndices = {
+    0, 1, 1};  // kThreadIndices[i] is the index of the thread that timer i corresponds to.
+
+const std::array<TimerInfo, kNumTimers> kTimers = []() {
```

Comment:
Nit/personal preference (feel free to ignore): With all those lambdas in the assignments of the constants, I would consider wrapping them in a test fixture class.